kinetic/src/main/java/org/kin/kinetic/KineticSdkInternal.kt

```diff
@@ -43,12 +44,12 @@ class KineticSdkInternal(
         apiHeaders += headers
         apiHeaders.put("kinetic-environment", environment)
         apiHeaders.put("kinetic-index", index.toString())
-        apiHeaders.put("kinetic-user-agent", "Kinetic Android SDK v0.0.1-rc.1")
+        apiHeaders.put("kinetic-user-agent", "Kinetic Android SDK v1.0.0-rc.3")
```

Comment:
I have a constants file with the name and version. I'll use those values here

Reply:
Nice! And it looks like you also had a helper method defined like in the TS SDK, which it's now using.